pkg/build/strategies/sti/sti.go

```diff
@@ -367,7 +367,7 @@ func (b *STI) Execute(command string, request *api.Request) error {
 				}
 				break
 			}
-			if glog.V(2) || command == api.Usage {
+			if glog.V(2) || request.AssembleOutput == true || command == api.Usage {
```

Comment:
shouldn't you also confirm that the command being run is assemble? As this is written you're going to get output from other Execution steps than just assemble, when assembleoutput is true, right?  eg save-artifacts